sdk/compute/arm-compute/package.json

```diff
@@ -3,8 +3,10 @@
   "sdk-type": "mgmt",
   "author": "Microsoft Corporation",
   "description": "A generated SDK for ComputeManagementClient.",
-  "version": "30.0.0-beta.3",
-  "engines": { "node": ">=12.0.0" },
+  "version": "30.0.0-beta.4",
+  "engines": {
```

Comment:
@qiaozha These changes must be due to the formatter. Please log an issue in the code gen repo to avoid this noise the next time you regenerate the package